backend/app/tests/ui/pages/home.py

```diff
@@ -19,11 +19,154 @@ def click_create_new_job_button(self):
+        self.pdf_git_radio_button.click()
+
+    @property
+    def web_preview_git_radio_button(self):
+        return self.page.locator("div.v-radio.git-preview-radio-button")
+
+    def click_web_preview_git_radio_button(self):
+        self.web_preview_git_radio_button.click()
+
+    @property
+    def modal_hint_text(self):
+        return self.page.locator(
+            "div.v-dialog__content.v-dialog__content--active > div > div > div.v-card__text"
+        )
+
+    @property
+    def collection_id_field_texts(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(1) > div > div > div.v-text-field__details"
+        )
+
+    @property
+    def version_field_texts(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(2) > div > div > div.v-text-field__details"
+        )
+
+    @property
+    def style_field_texts(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(3) > div > div > div.v-text-field__details"
+        )
+
+    @property
+    def content_server_field_texts(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(4) > div > div > div.v-text-field__details"
+        )
+
+    @property
+    def collection_id_field(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(1) > div > div > div.v-input__slot > div > label"
+        )
+
+    def fill_collection_id_field(self, value):
+        self.collection_id_field.fill(value)
+
+    @property
+    def version_field(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(2) > div > div > div.v-input__slot > div > label"
+        )
+
+    def fill_version_field(self, value):
+        self.version_field.fill(value)
+
+    @property
+    def style_field(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(3) > div > div > div.v-input__slot > div.v-select__slot > label"
+        )
+
+    @property
+    def style_field_drop(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(3) > div > div > div.v-input__slot > div.v-select__slot"
+        )
+
+    def fill_style_field(self, value):
+        self.style_field.fill(value)
+
+    def click_style_field(self):
+        self.style_field_drop.click()
+
+    @property
+    def style_field_dropdown(self):
+        return self.page.locator("#list-item-209-0 > div > div")
+
+    @property
+    def server_field(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(4) > div > div > div.v-input__slot > div.v-select__slot > label"
+        )
+
+    def fill_server_field(self, value):
+        self.server_field.fill(value)
+
+    @property
+    def content_server_locator(self):
+        return self.page.locator(
+            "div:nth-child(2) > div:nth-child(4) > div > div > div.v-input__slot > div.v-select__slot > label"
```

Comment:
With the next update to the CORGI UI we may want to work on some of these locators. I prefer the ability use `class_id` or `id` identifiers to simplify the important locators used for testing. It'll make the tests flaky any time there are changes to the UI. We're planning on having a bunch of those coming up!

Reply:
completely agree. In the meantime, I will check for better ones and replace, If I can...